<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetPasswordPage</class>
 <widget class="QWidget" name="ResetPasswordPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fourohfour Files - Reset Password</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="headingBarFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QHBoxLayout" name="headingBarLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="appLogoLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#2196F3;&quot;&gt;fourohfour&lt;/span&gt;&lt;span style=&quot; font-size:18pt; color:#424242;&quot;&gt; Files&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="headingBarSpacer">
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
            background-color: #757575; /* Grey button */
            color: white;
            border: none;
            border-radius: 4px;
            font-size: 11pt;
        }
        QPushButton:hover {
            background-color: #616161;
        }
        QPushButton:pressed {
            background-color: #424242;
        }</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="topContentSpacer">
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#2196F3;&quot;&gt;Reset &lt;/span&gt;&lt;span style=&quot; font-size:36pt; color:#424242;&quot;&gt;Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="subtitleLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#757575;&quot;&gt;We recommend using a password that is at least 15 characters long and unique.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="midContentSpacer">
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="formLayout">
     <item>
      <spacer name="leftFormSpacer">
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="inputLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <widget class="QLabel" name="oldPasswordLabel">
         <property name="styleSheet">
          <string notr="true">color: #424242;</string>
         </property>
         <property name="text">
          <string>Old Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="oldPasswordLineEdit">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 1px solid #E0E0E0;
    border-radius: 4px;
    padding: 8px;
    background-color: white;
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="newPasswordLabel">
         <property name="styleSheet">
          <string notr="true">color: #424242;</string>
         </property>
         <property name="text">
          <string>New Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passwordLineEdit">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 1px solid #E0E0E0;
    border-radius: 4px;
    padding: 8px;
    background-color: white;
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="confirmPasswordLabel">
         <property name="styleSheet">
          <string notr="true">color: #424242;</string>
         </property>
         <property name="text">
          <string>Confirm New Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="confirmPasswordLineEdit">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 1px solid #E0E0E0;
    border-radius: 4px;
    padding: 8px;
    background-color: white;
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showPasswordButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: #2196F3;
    border: none;
    background: transparent;
    text-align: left;
    padding: 0;
}</string>
         </property>
         <property name="text">
          <string>Show</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="buttonSpacer">
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="updatePasswordButton">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #2196F3;
    color: white;
    border: none;
    border-radius: 4px;
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>Update Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="rightFormSpacer">
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="bottomContentSpacer">
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
